integration-tests/utils/integration-utils.js

```diff
@@ -182,15 +182,37 @@ const delay = ms => {
 }
 
 const dismissNotifications = async page => {
-  await delay(3000) // delayed for any alerts to show
-  const notificationCloseButtons = await page.$x(
-    '(//a | //*[@role="button"] | //button)[contains(@aria-label,"Dismiss") and not(contains(@aria-label,"error"))]')
+  const notificationCloseButtonXpath = '(//a | //*[@role="button"] | //button)[contains(@aria-label,"Dismiss") and not(contains(@aria-label,"error"))]'
+  await Promise.race([
+    delay(3000), // delayed for any alerts to show
+    page.waitForXPath(notificationCloseButtonXpath, { visible: true })
+  ])
 
+  const notificationCloseButtons = await page.$x(notificationCloseButtonXpath)
   await Promise.all(
-    notificationCloseButtons.map(handle => handle.click())
+    notificationCloseButtons.map(button => button.evaluateHandle(btn => btn.click(), button))
   )
+  return !!notificationCloseButtons.length && delay(1000)
+}
 
-  return !!notificationCloseButtons.length && delay(1000) // delayed for alerts to animate off
+const dismissSurvey = async page => {
+  let element
+  try {
+    element = await page.waitForXPath('//iframe[@aria-label="NPS Survey"]', { timeout: 1000 })
+  } catch (e) {
+    // NPS survey was not found
+    return
+  }
+  try {
+    console.log('dismissing NPS survey')
+    const iframe = await element.contentFrame()
+    const [closeButton] = await iframe.$x('.//*[@id="ask-me-later"]')
```

Comment:
Can we use the button text instead of the ID? Not a huge deal, but it is our general pattern to only use things that are available to users (either through visible text or aria labels).